Tasks/Gradle/CodeAnalysis/Common/PmdTool.ts

```diff
@@ -62,10 +67,10 @@ export class PmdTool extends BaseTool {
     }
 
     /**
-    * Report parser that extracts the number of affected files and the number of violations from a report
-    *
-    * @returns a tuple of [affected_file_count, violation_count]
-    */
+     * Report parser that extracts the number of affected files and the number of violations from a report
```

Comment:
This does not seem to be the case in PmdTool.ts, which fully duplicates all documentation found in BaseTool.ts